src/main/java/de/tum/cit/aet/artemis/iris/service/pyris/dto/lectureingestionwebhook/PyrisLectureIngestionStatusUpdateDTO.java

```diff
@@ -7,5 +7,5 @@
 import de.tum.cit.aet.artemis.iris.service.pyris.dto.status.PyrisStageDTO;
 
 @JsonInclude(JsonInclude.Include.NON_EMPTY)
-public record PyrisLectureIngestionStatusUpdateDTO(String result, List<PyrisStageDTO> stages) {
+public record PyrisLectureIngestionStatusUpdateDTO(String result, List<PyrisStageDTO> stages, Long id) {
```

Comment:
```suggestion
public record PyrisLectureIngestionStatusUpdateDTO(String result, List<PyrisStageDTO> stages, long id) {
```
Can we use the base type here?